### ISSUE TITLE
Code is not correspond with this paper, and there are several errors.

### ISSUE THREAD

**Issue by ChenYuhang243** [dancsalo/TensorFlow-MIL#11]
1. The original paper used a joint cross-entropy loss, and codes in this repository don't show this.
2. For reasons I still don't know, train-loss and valid loss remain high and don't decrease after using this model.
